samtranslator/schema/schema.json

```diff
@@ -250245,6 +250245,20 @@
         "DependsOn": {
           "$ref": "#/definitions/PassThroughProp"
         },
+        "IgnoreGlobals": {
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            }
+          ],
+          "title": "Ignoreglobals"
```

Comment:
nit: can we capitalize the G in globals to keep consistent with the other titles

Reply:
This is auto-generated so I can't control how the schema generates.